core/aggregator/netsample/sample.go

```diff
@@ -90,38 +90,73 @@ func (s *Sample) SetUserDuration(d time.Duration) {
 	s.setDuration(keyRTTMicro, d)
 }
 
+func (s *Sample) GetUserDuration() int {
```

Comment:
The `Time` and `Duration` getter names should be `Get...Milliseconds` because the getter returns a different type than the setter.